libdnf/hy-package.cpp

```diff
@@ -914,6 +914,22 @@ dnf_package_get_supplements(DnfPackage *pkg)
     return reldeps_for(pkg, SOLVABLE_SUPPLEMENTS);
 }
 
+/**
+ * dnf_package_get_installed_requires_pre_post
+ * @pkg: a #DnfPackage instance.
+ *
+ * Gets the pre and post requires for an installed package.
```

Comment:
IMO being as descriptive and precise as possible helps especially with these niche and odd-behaving functions, how about this doc string?
> Returns Requires(pre) and Requires(post) dependencies of the package if it is installed. For a package that isn't installed returns an empty list.